.buildkite/_forge.rayci.yml

```diff
@@ -3,8 +3,20 @@ steps:
   - name: forge
     wanda: ci/docker/forge.wanda.yaml
 
-  - name: manylinux
+  - name: forge-aarch64
+    wanda: ci/docker/forge.aarch64.wanda.yaml
+    instance_type: builder-arm64
+
+  - name: manylinux-x86_64
```

Comment:
can we keep this as `manylinux`? and just add the other one? maybe just make a copy of the wanda file?

Reply:
I keep them explicit on purpose to avoid having to deal with them down the line with if-else like; having special naming for default cases is pretty troublesome

if arch == DEFAULT:
  docker = manylinux
else:
  docker = f"manylinux-{arch}"

Reply:
discussed offline, decided to drop the suffix. most of the ci/cd runs as `x86_64` today. if we add the suffix, we probably should add suffix to everything..

in the long term, if `aarch64` has more usage, the ci/cd toolchain/platform can handle this seemlessly (at rayci/wanda level)